webapp/security/views.py

```diff
@@ -1,6 +1,6 @@
 # Standard library
```

Comment:
On line `278` we can remove.
```
    if "isummary" in data:
        notice.isummary = data["isummary"]
```

Reply:
didn't I? I'm so confused. i thought I did.

Reply:
You removed it on one of the endpoints.